github/github_test.go

```diff
@@ -0,0 +1,60 @@
+package github
```

Comment:
I'd expect that the public functions are tested here: `FetchIssues`, `MoveIssues`, `ModifyPrepend`.

It's hard to do in a unit test because you don't want to actually call out to github. One way to do that is to have an interface for the actual execution of the commands. Then in the tests, the interface would be implemented to only record the commands that _would_ have been run. Then you could assert against those.